### ISSUE TITLE
homework

### ISSUE THREAD

**Issue by aceeccc** [Statistical-Connectomics-Sp15/intro#113]
Does anyone want to meet up sometime this weekend and do the most recent homework? I'm a bit confused with the last part of the lecture and would prefer to have someone to bounce ideas around with.

**Comment by DSP137** [comment]
I am not sure if you were able to actually able to meet up with anyone this time, but in the future, this is something I'd be interested in.

**Comment by aceeccc** [comment]
Nobody responded, so I haven't for this time :(

**Comment by dlee138** [comment]
Have you finished the HW yet?  I've been a bit behind on the HWs since I was mostly out of town last week and would like someone to work with.

**Comment by aceeccc** [comment]
No, I'll be at the Hodson 2nd floor area after 1 though, if you are in the area then.

**Comment by DSP137** [comment]
I've been struggling with this assignment too. Interested in setting up a regular time to meet up to work on homework and discuss topics from class?

**Comment by aceeccc** [comment]
Definitely. We can talk after class and find a time that works?
